apps/examples/App.tsx

```diff
@@ -133,27 +150,27 @@ const MainScreen = ({ navigation }: MainScreenProps): React.JSX.Element => (
     <Text style={styles.label} testID="root-screen-examples-header">
```

Comment:
I wonder about adding examples.length for consistency here

Reply:
https://github.com/software-mansion/react-native-screens/pull/2173/commits/5654ac07ae8004ca2c8d3a88531a36b9e29bfaa1